src/components/CameraHandler/index.js

```diff
@@ -200,14 +225,32 @@ class CameraHandler extends React.Component<Props, State> {
               />
             ) : <NoPermission requestPermission={this.requestPermission} />
           }
-        {currentQueueIndex !== null && !!queues && !!queues.length && (
-          <QueuePicker
-            queues={queues}
-            currentQueueIndex={currentQueueIndex}
-            onQueuePick={this.props.selectQueue}
-          />
+        {currentQueueIndex !== null
+          && currentQueueIndex !== undefined
+          && !!queues
+          && !!queues.length && (
+            <QueuePicker
+              queues={queues}
+              currentQueueIndex={currentQueueIndex}
+              onQueuePick={this.props.selectQueue}
+            />
         )}
         {shooting && <PhotoLoader />}
+        {sizeLimitExceeded && (
+          <View
+            pointerEvents="none"
+            style={{
+              ...StyleSheet.absoluteFillObject,
+              alignItems: 'center',
+              padding: 10,
+            }}
+          >
+            <Message
```

Comment:
Why did you decided to separate this message from the rest of message logic?

Reply:
I'd prefer if there was only one way of Message management

Reply:
just for the record :) I wanted the message to be displayed permanently. I will move the logic out of the component and use the same way for displaying the Message as we agreed